src/functions/template-tags/general.php

```diff
@@ -844,11 +844,11 @@ function tribe_event_featured_image( $post_id = null, $size = 'full', $link = tr
 			$post_id = get_the_ID();
 		}
 
-		$image_html     = get_the_post_thumbnail( $post_id, $size );
+		$image_html     = get_the_post_thumbnail( $post_id, apply_filters( 'tribe_event_featured_image_size', $size ) );
 		$featured_image = '';
 
 		//if link is not specifically excluded, then include <a>
-		if ( ! empty( $image_html ) && $link ) {
+		if ( ! empty( $image_html ) && apply_filters( 'tribe_event_featured_image_link', $link ) ) {
```

Comment:
Again, it would be appreciated if we could add inline docs for this new hook :smile: